plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/color-picker.spec.js

```diff
@@ -412,105 +412,105 @@ test.describe( 'Assembler -> Color Pickers', { tag: '@gutenberg' }, () => {
 		await expect( colorPicker ).toHaveClass( /is-active/ );
 	} );
 
-	test( 'Selected color palette should be applied on the frontend', async ( {
-		assemblerPageObject,
-		page,
-		baseURL,
-	} ) => {
-		const assembler = await assemblerPageObject.getAssembler();
-		const colorPicker = assembler
-			.locator(
-				'.woocommerce-customize-store_global-styles-variations_item'
-			)
-			.last();
-
-		await colorPicker.click();
-
-		await assembler.locator( '[aria-label="Back"]' ).click();
+	test(
+		'Selected color palette should be applied on the frontend',
+		{ tag: '@skip-on-default-pressable' },
```

Comment:
[Number of introductory patterns not same on Pressable](https://github.com/woocommerce/woocommerce/pull/51004/commits/83d5416a65ba76498fb7d94aaa86781767d3e034)